ansible/inventory.yml

```diff
@@ -131,6 +134,11 @@ hosts:
         win10-arm64-1: { vs: '2017' }
         win10-arm64-2: { vs: '2017' }
 
+    - ibm:
+        aix71-ppc64_be-1: {ip: 129.33.196.197, user: b9s010a}
+        aix71-ppc64_be-2: {ip: 129.33.196.198, user: b9s010a}
```

Comment:
Should we attempt to keep providers sorted alphabetically? (It isn't quite before this change but almost is.)

Reply:
I agree